code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -225,17 +225,22 @@
 
 /obj/item/clothing/head/helmet/space/plasmaman/atmospherics
 	name = "atmospherics plasma envirosuit helmet"
-	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue."
+	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue. Has thermal shielding." // Added the thermal shielding clarification
```

Comment:
```suggestion
	desc = "A space-worthy helmet specially designed for atmos technician plasmamen, the usual purple stripes being replaced by atmos' blue. Has improved thermal shielding." // Added the thermal shielding clarification
```
The normal space suits already have thermal shielding but not the atmos helmet level.

Reply:
I mean, yeah, but the other space suits don't have a "Has thermal shielding line" like the atmos one has. I think it's better to keep it like this for the sake of consistency with other descriptions

Reply:
Yes but having improved or another word to point out it is as good as the hardsuit halmet helps the players. Helmets do have a little line on examine on how much fire protection they give tho so either works.

Reply:
Eh, I changed my mind about that, your proposal's better.